src-electron/generator/helper-zcl.js

```diff
@@ -2693,6 +2699,83 @@ function if_compare(leftValue, rightValue, options) {
   }
 }
 
+/**
+ * Check if the given type is signed or not based on the type name and cluster
+ * id.
+ * @param {*} type
+ * @param {*} clusterId
+ * @param {*} options
+ * @returns Promise of content
+ */
+async function if_is_data_type_signed(type, clusterId, options) {
```

Comment:
Again, is it better to use id or cluster code here?  For purposes of doing this in a loop over clusters it doesn't matter much, but should this be usable outside such a loop?

Reply:
id is unique so prefer using id. code is unique in Matter but not unique in zigbee pro. This way it is used in a generic way.